### DISCUSSION TITLE
How do I increase the resolution of image rendering?

### DISCUSSION THREAD

**Discussion by xtiondiv123** [lllyasviel/Fooocus#2829] [General]
Hi， this is my question about the final picture resolution. 
![2024-04-26_16-50-24_7266](https://github.com/lllyasviel/Fooocus/assets/152345887/3fff2898-49ed-4475-82e6-119c487b32d7)
The floor part is unchanged, the part of the scene that is rendered. But the resolution of the scene part is very low.
 Thanks for reply.

**Comment by ILikeToasters** [comment]
I am not sure what you are asking?

**Comment by mashb1t** [comment]
@xtiondiv123 i also don't get what you're referring to. If you're trying to increase the resolution and details in general please check out the upscale or vary tab in image prompt.

**Comment by Rogala** [comment]
If your drawing is a single piece and not modified at the bottom through Inpaint or Outpaint, the quality problem may be due to incorrect settings and components.

1. After selecting Performance you get the initial settings with which the program works well (red marker).
2. Then you could select Refiner and specify its settings, which are not always correct (blue marker).
3. Next you added LoRA 2-5 and their settings, they may not be correct either, and some may not work with this Base Model or in speed mode (yellow marker).

Try the same Promt but with the lowest number of settings, then increase them as needed.

![Снимок экрана 2024-04-28 103503](https://github.com/lllyasviel/Fooocus/assets/7437582/538073e2-c1e9-498a-8715-fd7180468729)

**Reply by mashb1t** [reply]
@Rogala thanks for the analysis
@xtiondiv123 keep in mind that lightning performance will not produce high-quality images compared to speed or quality

**Reply by xtiondiv123** [reply]
hi， Rogala thanks for your reply。
Most of my application environment is  building space, original photo need redraw . My Fooocus is Chinese version maybe you mean Inpaint . so if I have multiple button permutations and combinations to debug, there are many possibilities for this operation. So can you help give a reasonable argument by building the requirements of the space? Thank you very much.

**Reply by BNP1111** [reply]
I think maybe what you might have meant might be this, parts of the image are blurry and need to be improved in quality, so what you do is, if the part of the area you want to modify is not high resolution, then you cut out that area and zoom in, or zoom in on the whole image and then Inpainting (low denoising) the part of the area, and then zoom out and paste it back in, and then maybe that's the answer you're looking for

**Reply by xtiondiv123** [reply]
hi，BNP. you are right， my original picture is photo. The resolution is great. when I change the build space  which part is low resolution. And i want to change the part of rebuilding to high resolution. Like Rogala said，I should change the lora  and modify some parameter.
So you mean another way for use the low resolution picture then increasing resolution in other software？ This software only increasing resolution. Thanks you.
